### PR TITLE
Ruc/header search bar fixes

### PR DESCRIPTION
**Pull request by whalay**
I made the changes specified in the review.
localhost_3000_ (1)
localhost_3000_ (1)

### CODE REVIEW

**Comment by abdulmaleekadams** [comment]
@aftabrehan  Kindly review

File: src/routes/card-payment/payment.css [hunk 1]
```diff
@@ -1,164 +1,164 @@
...
 }
 
 .flexColumn {
-  flex-direction: column;
+    flex-direction: column;
 }
 
 .w-100 {
-  width: 100%;
+    width: 100%;
```
Comment (mrxshinji) [inline src/routes/card-payment/payment.css:10]: This part is not fixed yet

**Review by mrxshinji — CHANGES_REQUESTED** [review]
Changes requested not complete

**Review by mrxshinji — APPROVED** [review]

**Review by aftabrehan — APPROVED** [review]
Excellent PR, @whalay 💪 💯 🚀

**Pull request merged**